### ISSUE TITLE
Forced update not compatible with 10.11 without warning and without allowing to keep the compatible version

### ISSUE THREAD

**Issue by MacTerrassa** [sprout2000/leafview#315]
I'm retesting your app since I deleted it a long time ago due to compatibility issues with my 10.11 machine. I have tried the latest version, which requires 10.13. I reinstall version 1.9.8 as the icon seems to indicate that it is still supported. But when you open it, without any incompatibility alert, it forces you to update to the new one, which, as I say, requires 10.13 and does not indicate anything. It forces you to update knowing that it is not compatible with your system. Because? If my installation is invalid, I understand that I should be able to continue using the previous version and not force you to upgrade without notice. 

On the other hand, I have not seen on your page the slightest indication about those requirements or their changes over time. Wouldn't it be interesting to clearly indicate them, and mark such a final version compatible with such a system, and make it easier for the possible user to find the versions? 

Some time ago, I found that it was supposed to be compatible with 10.11 but the program didn't start, I had to go back to a previous version that did work but I still upgraded to one that didn't, but since it supposedly "should" be, it seemed like a bug but with some logic. Now that it's compiled for new systems, it doesn't make any sense to install it. Thus, what he achieves is that users stop trying his program.

**Comment by sprout2000** [comment]
@MacTerrassa 

Due to limited resources for personal development, it is not possible to support versions for which Apple has already discontinued support.
If you want to use the latest version of _LeafView_ on v10.11, you have to build this project for yourself.

```sh
$ git clone git@github.com:sprout2000/leafview.git
$ cd leafview
$ npm install
$ npm run build && npm run package
```

_NOTE: You will need to have [Node.js]() and [Command Line Tools](https://developer.apple.com/download/all/) installed._

And since v1.10.6, auto-update feature has been disabled, IIRC.
Sorry for the inconvenience.

**Comment by MacTerrassa** [comment]
I understand that supporting old systems is complicated and not possible in many cases, especially for small developers. But not so much that automatic updates end up installing incompatible versions. And I would also like those requirements to be clear, and when they change the documentation and notices would also be updated. I accept that the program warns of an incompatible version and that if you want to install it you have to update your system first, but always indicating what it needs and what it doesn't to be able to actually install it. 
Thanks for your reply. Regards. 
MacTerrassa

**Comment by sprout2000** [comment]
@MacTerrassa 

I have released updated versions of past versions as [`v1.10.7`](https://github.com/sprout2000/leafview/releases/tag/v1.10.7), [`v1.9.9`](https://github.com/sprout2000/leafview/releases/tag/v1.9.9), [`v1.8.20`](https://github.com/sprout2000/leafview/releases/tag/v1.8.20), [`v1.7.15`](https://github.com/sprout2000/leafview/releases/tag/v1.7.15), [`v1.6.9`](https://github.com/sprout2000/leafview/releases/tag/v1.6.9) and [`v1.4.7`](https://github.com/sprout2000/leafview/releases/tag/v1.4.7).
I don't know exactly which version will work with past macOS versions, but you can try them if you like.

_Note that these versions have been updated to be able to reject auto updates._

Thank you for your interest in my project. Best regards.
